assets/base.css

```diff
@@ -1492,11 +1492,11 @@ button.shopify-payment-button__button--unbranded {
 .share-button__button {
   font-size: 1.4rem;
   display: flex;
-  min-height: 2.4rem;
   align-items: center;
   color: rgb(var(--color-link));
   margin-left: 0;
   padding-left: 0;
+  height: 4.4rem;
```

Comment:
I think it might be a better approach to make it `min-height: 4.4rem`. I see that it's not the approach we used for the `pop up link` block but we should probably change it there too 🤔 

The reason I say that is the fact that you could have a longer text than just `share` whether it's in another language or something the merchant edits. Though it's less likely with `share` than it is with the `pop up link` text. 
And if the content is bigger then it outgrows the button in a weird way: 
<details>
<summary>Screenshot</summary>

![](https://screenshot.click/15-30-tc5td-yrada.png)
</details>

Reply:
Thanks Ludo, good catch. I changed `height` to `min-height` for both **share-button** and **pop-up**. I also changed `width` to `min-width` for `.icon-share` to prevent shrinking in case when content getting bigger 
<details>
<summary>Screenshot</summary>
<img width="368" alt="Screenshot 2022-11-17 at 11 17 50 AM" src="https://user-images.githubusercontent.com/105315663/202539106-d1f5214f-0871-414f-859c-8161c660a17f.png">
</details>